README.md

```diff
@@ -178,6 +178,85 @@ For the information on how to run tests and configure them, go to the [`tests`](
    docker run --rm -it -p 3001:3001 -v <path to your custom config>:/app/core-ui/environments/ --env ENVIRONMENT={your-env} --pid=host --name busola europe-docker.pkg.dev/kyma-project/prod/busola:latest
    ```
 
+## Deploy Busola on the Kubernetes Cluster
+
+To install Busola on the Kubernetes cluster, run:
+
+```shell
+(cd resources && kustomize build base/ | kubectl apply -f- )
+```
+
+To install Busola using a specific environment configuration, set the `ENVIRONMENT` shell environment variable and run:
+
+```shell
+(cd resources && kustomize build environments/${ENVIRONMENT} | kubectl apply -f- )
+```
+
+## Access Busola Installed on Kubernetes
+
+### kubectl
+
+The simplest method that always works is to use the capabilities of kubectl.
+
+```shell
+kubectl port-forward services/busola 3001:3001
+```
+
+### k3d
+
+Prerequisites:
+
+- K3d with exposed loadbalancer on port 80.
+  > **TIP:** To create K3d with exposed load balancer run: `k3d cluster create -p "80:80@loadbalancer"`.
+  > More details available in k3d docs [exposing services](https://k3d.io/v5.6.3/usage/exposing_services/).
+
+1. Install Ingress resources:
+
+```shell
+(cd resources && kubectl apply -f ingress/ingress.yaml)
+```
+
+2. Go to `localhost`
```

Comment:
Now that I think of it. What does it mean?
Go to the localhost folder? Something else?